src/modules/todo.js

```diff
@@ -0,0 +1,110 @@
+const form = document.querySelector('.form');
+const input = document.querySelector('.todoInput');
+const list = document.querySelector('.todoList');
+const clearComplete = document.querySelector('.clear-btn');
+
+let todoItems = [];
+
+function renderTodo(todo) {
+  localStorage.setItem('todoItems', JSON.stringify(todoItems));
+
+  const item = document.querySelector(`[data-key='${todo.index}']`);
+
+  if (todo.deleted) {
+    item.remove();
+    if (todoItems.length === 0) list.innerHTML = '';
+    return;
+  }
+
+  const iscompleted = todo.completed ? 'done' : '';
+  const node = document.createElement('li');
+  node.setAttribute('class', `todo-item ${iscompleted}`);
+  node.setAttribute('data-key', todo.index);
+  node.innerHTML = `
+    <input index="${todo.index}" type="checkbox"/>
+    <label for="${todo.index}" class="tick js-tick"></label>
+    <input value="${todo.description}" class="todo" ></input>
+  
+    <i class="fa-solid fa-ellipsis-vertical js-delete-todo"></i>
+   
+  `;
+
+  if (item) {
+    list.replaceChild(node, item);
+  } else {
+    list.append(node);
+  }
+}
+function addTodo(description) {
```

Comment:
- [ ] [optional] The index of the tasks is dos not start from 1 when adding a new task to the list, please consider fixing this so your app can work as expected

Reply:
@Ismailco thanks